README.md

```diff
@@ -84,6 +84,8 @@ will be cached and it will not take so long.
  ctgov  | conditions                 | table | postgres
  ctgov  | countries                  | table | postgres
 ```
+12. Create view "basic_search" in aact database. View definition is available in the file backend/database_scripts/basic_search.sql.
+Command to run: psql -U postgres -d aact {insert statements from basic_search.sql}
```

Comment:
* I'm confused what exactly is the command you would run here. What is "statements from basic_search.sql" ? The instructions should be easy to follow for someone who has no experience.
* Also the instruction should have continuity with the previous instructions (you are already in psql and connected to the aact database so running the command `psql -U postgres` wouldn't work.), or just put this in a separate list of instructions.
* Also, it's good to use backticks in the README to make the command stand out.